.github/workflows/auto-label-issue.yml

```diff
@@ -0,0 +1,25 @@
+name: Auto Label New Issues
+
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  issues: write
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v3
+
+    - name: Add Labels to Issues
+      if: github.event_name == 'issues'
+      uses: actions-ecosystem/action-add-labels@v1.1.1
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        labels: |
+          hacktoberfest
+          hacktoberfest-accepted
```

Comment:
Thanks for your contribution! I just wanted to ask if you've tested the workflow on your fork before submitting the PR. This will help ensure that everything runs smoothly once merged.

Reply:
![image](https://github.com/user-attachments/assets/62d19569-d86c-494a-b564-b8d8982fc6e2)
Just tested and it worked